### PR TITLE
doc: add examples for streamnative platform

### PR DESCRIPTION
**Pull request by rockybean**

### CODE REVIEW

**Review by Jennifer88huang — APPROVED** [review]

**Review by Huanli-Meng — COMMENTED** [review]
leave my comments. PTAL

File: platform/values_cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+  volumes:
+    useSeparateDiskForTxlog: true
+    data:
+      name: data
+      size: 10Gi
+      storageClassName: local-ssd
+    dataLog:
+      name: datalog
+      size: 10Gi
+      storageClassName: local-hdd
```
Comment (Huanli-Meng) [inline platform/values_cluster.yaml:82]: remove storageclassname

**Comment by Huanli-Meng** [comment]
add the YAML file of the local volume provisioner to this example

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Pull request merged**